testutils/testutils.go

```diff
@@ -0,0 +1,55 @@
+package testutils
+
+import (
+	"context"
+	"database/sql"
+	_ "modernc.org/sqlite"
+	"testing"
+)
+
+func GetEmptyDb(t *testing.T) *sql.DB {
+	t.Helper()
+	db, err := sql.Open("sqlite", ":memory:")
```

Comment:
Seems like SQLite is already paying dividends 😆 
That's nice.

In the (very) long run, it's debatable whether it's good to mainly test on in-memory DBs when PostgreSQL is still just an important.

But it's great to have any actual tests for the SQL parts to begin with, so: 👍🏻

Reply:
Absolutely! I was thinking exactly this when discovering the issues #7 and #9! 

I would argue that we should still consider postgres as our "main" database, mostly because sqlite has concurrency issues by design, and every http requests runs in it's own goroutine.  I'd expect postgres to perform significantly better in that regard.